drivers/i2c/Kconfig

```diff
@@ -22,6 +22,12 @@ config I2C_SHELL
 	  The I2C shell supports scanning, bus recovery, I2C read and write
 	  operations.
 
+config I2C_ZERO_LEN_XFER_SUPPORTED
```

Comment:
If this config use only in i2c_sell.c, the name should be better start with I2C_SHELL_.